compiler/onecc-docker/onecc-docker

```diff
@@ -51,7 +51,7 @@ def main():
         response = requests.get(LATEST_URL)
         response.raise_for_status()
     except requests.exceptions.RequestException as e:
-        SystemExit('onecc-docker: error: {}'.format(e))
+        raise SystemExit('onecc-docker: error: {}'.format(e))
```

Comment:
https://github.com/Samsung/ONE/pull/9800#discussion_r980901883 

I checked our mistake and added `raise`.